style.css

```diff
@@ -417,6 +427,75 @@ textarea::placeholder {
   border-radius: 8px;
 }
 
+.bkg__popup {
+  padding: 1rem;
+  backdrop-filter: blur(10px);
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  overflow-y: scroll;
+}
+
+#popup {
+  background-color: white;
+  padding: 1rem;
+  border-radius: 10px;
+}
```

Comment:
- [ ] It is required to stick to the Figma design as close as possible. The width of your popup box on the desktop is wider in oppose to what we have on the guiding template. Kindly take a look at both designs and work your design alongside the guiding template.

### Your Design
![your-popup-desktop](https://user-images.githubusercontent.com/26597432/216720530-defbf9ea-4656-42c0-926d-46f03190705a.png)

### Figma Design
![temp-desktop](https://user-images.githubusercontent.com/26597432/216720555-b0850931-a1f6-405b-8ba5-86540099eff6.png)

- [OPTIONAL] You might also need to look into how the image and texts appear on your popup box. They look kind of blur. They don't look as clear as they are on the page itself. It would be good if you could take care of that too.

Reply:
Thanks again, I hadn't noticed... May I ask why you thought I was Abraham(Cabraham1)? It's okay, it just caught my attention.